index.html

```diff
@@ -419,7 +419,7 @@ <h4 class="card-header">Plotting tutorial</h4>
 
     <div class="col-lg-4 mb-4">
       <div class="card h-100">
-        <h4 class="card-header">Parallel documentation</h4>
+        <h4 class="card-header">Parallelization documentation</h4>
```

Comment:
`Paralellisation` in UK spelling? ;-)

Reply:
check - will be corrected. Thanks for pointing it out.